README.md

```diff
@@ -1,13 +1,40 @@
 # Project Happy Thoughts API
+This is a project built during week 19 in the Technigo Bootcamp. 
+The project was to build an API which includes both GET request endpoints to return data and POST endpoints to create data and use this to replace an API from the week 11 project with our newly built one. I built the API using Express and MongoDB. 
 
-Replace this readme with your own information about your project.
+Main goals were to learn: 
+- How to build a full API which includes handling of user input
+- How to include error handling to return good validation errors
+- How to build an API which works well with an existing frontend
 
-Start by briefly describing the assignment in a sentence or two. Keep it short and to the point.
+The existing frontend was a twitter-like page where the user should post only happy thoughts. It also shows a count of likes/hearts on each thought. 
 
 ## The problem
+- The setup for this Express API consists of a MongoDB stored in Atlas and deployed to Heroku 
+- Using Mongoose for creating endpoints and manipulating the data 
+- I have created one main thoughtSchema which defines the structure of the document, with default values for hearts and createdAt and with validators for the message. I have also added username with default value 'anonymous' in case the input for the username is left empty, and with maxlength of 20 characters.
 
-Describe how you approached to problem, and what tools and techniques you used to solve it. How did you plan? What technologies did you use? If you had more time, what would be next?
+I created the following endpoints:
+
+https://sofias-happy-thought-api.herokuapp.com
+
+GET /thoughts
+--> To get all thoughts. I used mongoose methods to sort from newest thougth to the oldest, and added .skip() and .limit() to add pagination. With this addition I also needed to update my frontend to make it work. As for now it is fully functioning, however it is possible to click past the last page with thoughts, which is something I will work on fixing later on. 
```

Comment:
Maybe you could in the backend calculate (from the total number of thoughts, page, and perPage) how many pages there are in total, and which one is being shown at the moment - and then just cook up some logic in the frontend that enables/disables the next/prev buttons depending on that and also shows how many pages there are ("showing page 1 of 6")